presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -452,30 +451,19 @@ private PlanBuilder appendApplyNode(
         return Optional.empty();
     }
 
-    private PlanBuilder createPlanBuilder(Expression expression)
+    private PlanBuilder createPlanBuilder(Node node)
```

Comment:
Maybe move `createPlanBuilder` method to `QueryPlanner` as static method? There is very similar method there: `planFrom`, which also does this magic:
```
        // Make field->symbol mapping from underlying relation plan available for translations
        // This makes it possible to rewrite FieldOrExpressions that reference fields from the FROM clause directly
        translations.setFieldMappings(relationPlan.getOutputSymbols());
```